### DISCUSSION TITLE
My account is blocked.

### DISCUSSION THREAD

**Discussion by KyrylHodlevskiy** [vercel/community#3785] [Account Issue] [answered]
### Question

I got message: " This user account is blocked." when i was trying to log in using Git Hub

**Comment by activw** [comment]
I got message: " This user account is blocked." when i was trying to log in using Git Hub

**Comment by amyegan** [comment, marked as answer]
Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. For more info, please check this guide: [Why has my account or deployment been paused?](https://vercel.com/guides/why-is-my-account-deployment-blocked)
